test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -275,7 +275,7 @@ export $envVarName='$guid'
         }
 
         It "Doesn't run the login profile when -Login not used" {
-            $result = & $powershell -Command "`$env:$envVarName"
+            $result = & $powershell -noprofile -Command "`$env:$envVarName"
```

Comment:
hmmm, this seems to invalidate the test - `-login` is not used, so `-noprofile` should not be needed.

Reply:
No, this existing test is validating that the login profile code path isn't executed if `-login` isn't specified.  `-noprofile` is orthogonal to `-login`.  I needed to add `-noprofile` here as my profile.ps1 was causing this test to fail.  This change is technically not part of my changes, but found it running the test locally.